### ISSUE TITLE
[Part 2] `nodejs-polars` fails on supported NAPI function

### ISSUE THREAD

**Issue by controversial** [oven-sh/bun#2056]
### What version of Bun is running?

0.5.7 (8b4e58f3d225e30d186fc2d208dea477a2f9b5a5)

### What platform is your computer?

Darwin 22.3.0 arm64 arm

### What steps can reproduce the bug?

The following script attempts to use the [`nodejs-polars`](https://www.npmjs.com/package/nodejs-polars) package, which is the official bindings (using napi-rs) to the [Polars](https://github.com/pola-rs/polars) dataframe library.

The following simple script attempts to create a dataframe:
```js
import pl from 'nodejs-polars';

const df = pl.DataFrame([
  { a: 1, b: 2 },
  { a: 4, b: 10 },
]);
console.log(df);
```

### What is the expected behavior?

Using Node:
```
❯ node test.js
shape: (2, 2)
┌─────┬──────┐
│ a   ┆ b    │
│ --- ┆ ---  │
│ f64 ┆ f64  │
╞═════╪══════╡
│ 1.0 ┆ 2.0  │
├╌╌╌╌╌┼╌╌╌╌╌╌┤
│ 4.0 ┆ 10.0 │
└─────┴──────┘
```

### What do you see instead?

Using Bun:
```
❯ bun test.js
125 |                 try {
126 |                     if (localFileExisted) {
127 |                         nativeBinding = require('./nodejs-polars.darwin-arm64.node');
128 |                     }
129 |                     else {
130 |                         nativeBinding = require('nodejs-polars-darwin-arm64');
                            ^
TypeError: dlopen(/Users/luke/Developer/dataset/node_modules/nodejs-polars-darwin-arm64/nodejs-polars.darwin-arm64.node, 0x0001): symbol not found in flat namespace '_napi_get_value_bigint_words'
      at /Users/luke/Developer/dataset/node_modules/nodejs-polars/bin/native-polars.js:130:24
      at /Users/luke/Developer/dataset/node_modules/nodejs-polars/bin/internals/polars_internal.js:6:27
      at /Users/luke/Developer/dataset/node_modules/nodejs-polars/bin/series/series.js:7:26
      at /Users/luke/Developer/dataset/node_modules/nodejs-polars/bin/index.js:24:15
      at /Users/luke/Developer/dataset/test.js:1:0
```

### Additional information

The [napi tracking issue](https://github.com/oven-sh/bun/issues/158) says `napi_get_value_bigint_words` should be supported.

This issue is an exact duplicate of #2053, for a different NAPI function.

**Comment by Jarred-Sumner** [comment]
> The https://github.com/oven-sh/bun/issues/158 says napi_get_value_bigint_words should be supported.

Yeah...I remember implementing this. But I don't see the code.

Edit: Oh, the one I remember was this which is not exactly the same

https://github.com/oven-sh/bun/blob/0d102d16ebf7dfaae4108bd8b60550b1cbc1dfca/src/bun.js/bindings/napi.cpp#L1599-L1628

**Comment by controversial** [comment]
If it’s not implemented, #158 should probably be updated?

**Comment by ThatOneBro** [comment]
I'll get this fixed really quick today. We should probably double check to make sure all of the list is up to date though
